draft-pardue-httpbis-identity-digest.md

```diff
@@ -296,8 +296,16 @@ Unencoded-Digest: \
 
 # Security Considerations
 
-The considerations in {{DIGEST-FIELDS}} apply. There are no known additional
-considerations.
+The considerations in {{DIGEST-FIELDS}} apply.
+
+HTTP messages can contain both Content-Encoding and Unencoded-Digest
+({{encoding-and-unencoded}}). In order to validate the Unencoded-Digest, encoded
+content would need to be decoded. The might offer the opportunity for an
+attacker to direct malicious data into a decoder. One possible mitigation would
+be to also provide a Content-Digest or Repr-Digest in the message, allowing for
+validation of the received bytes before further processing. Furthermore, the use
+of signatures as explained in {{Section 6.3 of DIGEST-FIELDS}} can address data
+substitution attacks.
```

Comment:
This assumes that the attacker can control the (encoded) body of the response, but not the headers, right? It might be worth specifying that as the threat model here.

This doesn't seem like an incredibly plausible attack vector, but it's certainly reasonable to point out the ways in which the one header can defend against an attack that the other couldn't.

Reply:
Yes your right.

The security considerations in rfc 9530 capture the entire shape of threat model, I was trying to not repeat that text and just highlight the difference introduced. Let me see if I can wordsmith a little to find a balance.